test/tests/operations/Image.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Image operation tests.
+ *
+ * @author tlwr [toby@toby.codes]
+ *
+ * @copyright Crown Copyright 2017
+ * @license Apache-2.0
+ */
+import TestRegister from "../../TestRegister.js";
+
+TestRegister.addTests([
+    {
+        name: "Extract EXIF: nothing",
+        input: "",
+        expectedOutput: "Found 0 tags.\n",
+        recipeConfig: [
+            {
+                op: "From Hex",
```

Comment:
Is this operation necessary to test the EXIF operation with no input? It's not required to convert the input to a `byteArray` - it will be converted implicitly.